src/Components/Components/src/Routing/Router.cs

```diff
@@ -80,17 +78,11 @@ public class Router : IComponent, IHandleAfterRender, IDisposable
         /// <summary>
         /// Gets or sets a flag to indicate whether route matching should prefer exact matches
         /// over wildcards.
+        /// <para>This property is obsolete and configuring it does nothing.</para>
         /// </summary>
-        /// <remarks>
-        /// <para>
-        /// Important: all applications should explicitly set this to true. The option to set it to false
-        /// (or leave unset, which defaults to false) is only provided for backward compatibility.
-        /// In .NET 6, this option will be removed and the router will always prefer exact matches.
-        /// </para>
-        /// </remarks>
         [Parameter] public bool PreferExactMatches { get; set; }
```

Comment:
Are we not removing it to prevent breaking changes?

Reply:
If we don't remove it, perhaps we could throw if its set to false?

Reply:
It's unusual to set this to false. In the ordinary case this would not appear in the user code and consequently default to false. Given we are going to be unable to do anything useful there, it seems less interesting to go out of the way to throw in the rare case.